pkg/cfg/env_test.go

```diff
@@ -0,0 +1,62 @@
+package cfg
+
+import (
+	"os"
+	"testing"
+
+	"github.com/google/uuid"
+)
+
+func TestEnvString(t *testing.T) {
+	key := uuid.NewString()
+	value := uuid.NewString()
+	notsetKey := uuid.NewString()
+	err := os.Setenv(key, value)
+	if err != nil {
+		t.Fatalf("failed to set environment variable: %v", err)
+	}
```

Comment:
suggestion: I would use `gotest.tools/v3/assert` here.
```suggestion
	assert.NilError(t, err, "failed to set environment variable")
```